manifests/vhost.pp

```diff
@@ -16,12 +16,15 @@
   Optional[String]             $content = undef,
 ) {
 
+  include caddy
```

Comment:
Defines should include their class if it is necessary for operation. This is our case, as we need caddy to be installed.